chapter04/pxd/ex2.pl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use feature "say";
+use Scalar::Util qw( looks_like_number );
+
+sub total {
+    chomp(my @vars=@_ );
+    my $total=0;
+
+    foreach my $number  (@vars) {
+        if ( looks_like_number ($number)) {
+            $total += $number;    
+        } else {
+            say " {$number} is not a numeric --> INGORING!! ";
+        }
+    }
+    return $total;
+}
+
+sub math_total {
```

Comment:
Cool maths! I think this only works for incremental sequential numbers though :)